web-app/vercel.json

```diff
@@ -21,5 +21,10 @@
       "path": "/api/sync/source-import",
       "schedule": "*/5 * * * *"
     }
+    ,
+    {
+      "path": "/api/sync/job-schedules",
+      "schedule": "*/5 * * * *"
```

Comment:
If we only run the jobs every 5 minutes, why can the user choose a time for each minute?

Reply:
I changed it to every minute for now, cause the user can define on the minute, then the job should run on the minute 769eebc2b69b0f0430f7033378fe8b76a6c149d2